BitmovinConvivaAnalytics/Classes/ConvivaAnalytics.swift

```diff
@@ -517,22 +513,18 @@ private extension ConvivaAnalytics {
         logger.debugLog(message: "Player state changed: \(playerState.rawValue)")
     }
 
-    private func handleAdUpdateRequest() {
-        guard let player, isAd else { return }
-
-        adAnalytics.reportPlaybackMetric(
-            CIS_SSDK_PLAYBACK_METRIC_PLAY_HEAD_TIME,
-            value: Int64(player.currentTime(.relativeTime) * 1_000)
-        )
-    }
-
     private func reportPlayHeadTime() {
         guard let player, isSessionActive else { return }
 
-        videoAnalytics.reportPlaybackMetric(
-            CIS_SSDK_PLAYBACK_METRIC_PLAY_HEAD_TIME,
-            value: Int64(player.currentTime(.relativeTime) * 1_000)
-        )
+        let reportPlayHeadTime: (_ analytics: CISStreamAnalyticsProtocol) -> Void = { analytics in
```

Comment:
I'm open for a discussion of introducing a custom protocol instead of using `CISStreamAnalyticsProtocol`. But it's there and it's public so I decided to use it directly.